esp8266-fastled-webserver/esp8266-fastled-webserver.ino

```diff
@@ -587,9 +741,11 @@ void loop() {
   // Call the current pattern function once, updating the 'leds' array
   patterns[currentPatternIndex].pattern();
 
-  FastLED.show();
```

Comment:
Because `FastLED.delay()` is guaranteed to call FastLED.show() at least once, this call is redundant.